crates/cdk-mintd/Cargo.toml

```diff
@@ -6,9 +6,14 @@ authors = ["CDK Developers"]
 license = "MIT"
 homepage = "https://github.com/cashubtc/cdk"
 repository = "https://github.com/cashubtc/cdk.git"
-rust-version = "1.63.0"                            # MSRV
```

Comment:
Removed by mistake?

Reply:
mintd doesn't have an msrv or at least not 1.63 as it pulls in lnd that had a higer msrv and redb